habitat-lab/habitat/tasks/nav/nav.py

```diff
@@ -1537,7 +1536,16 @@ def __init__(self, *args: Any, **kwargs: Any):
         super().__init__(*args, **kwargs)
 
         # Init goto velocity controller
-        self.w2v_controller = ContinuousController(self._config)
+        try:
+            from habitat.utils.controller_wrapper import ContinuousController
+
+            self.w2v_controller = ContinuousController(self._config)
```

Comment:
ContinuousController is a very vague name at this API level. Please change the name to a more descriptive one.

Reply:
Changed it to `DiffDriveVelocityController`. Let me know if there's a better one!